docs/src/pages/lab/slider/DisabledSlider.js

```diff
@@ -17,9 +17,9 @@ function DisabledSlider(props) {
 
   return (
     <div className={classes.root}>
-      <Slider classes={{ container: classes.slider }} value={0} disabled />
-      <Slider classes={{ container: classes.slider }} value={50} disabled />
-      <Slider classes={{ container: classes.slider }} value={100} disabled />
+      <Slider classes={{ root: classes.slider }} value={0} disabled />
```

Comment:
```suggestion
      <Slider className={classes.slider} value={0} disabled />
```
would be equivalent? I would prefer this solution just because it's simpler. The simpler the demo the better.

Reply:
yeah, just changed them to be like that :)